rest-api/src/test/java/gov/cms/qpp/conversion/api/services/DbServiceImplTest.java

```diff
@@ -92,13 +94,13 @@ void testNoWriteBecauseNoAudit() {
 
 	@Test
 	void testGetUnprocessedCpcPlusMetaData() {
-		PaginatedQueryList<Metadata> mockMetadataList = mock(PaginatedQueryList.class);
-		when(mockMetadataList.stream()).thenAnswer(invocationOnMock -> Stream.of(new Metadata(), new Metadata()));
-		when(dbMapper.query(eq(Metadata.class), any(DynamoDBQueryExpression.class))).thenReturn(mockMetadataList);
+		QueryResultPage<Metadata> mockMetadataPage = mock(QueryResultPage.class);
+		when(mockMetadataPage.getResults()).thenReturn(Lists.newArrayList(new Metadata(), new Metadata()));
+		when(dbMapper.queryPage(eq(Metadata.class), any(DynamoDBQueryExpression.class))).thenReturn(mockMetadataPage);
 
 		List<Metadata> metaDataList = underTest.getUnprocessedCpcPlusMetaData();
 
-		verify(dbMapper, times(Constants.CPC_DYNAMO_PARTITIONS)).query(any(Class.class), any(DynamoDBQueryExpression.class));
+		verify(dbMapper, times(Constants.CPC_DYNAMO_PARTITIONS)).queryPage(any(Class.class), any(DynamoDBQueryExpression.class));
 		assertThat(metaDataList).hasSize(2 * Constants.CPC_DYNAMO_PARTITIONS);
```

Comment:
2 is a magic number.  Either create a constant with a name that describes it, or add some documentation so we understand it.